packages/manager/modules/pci/src/projects/project/additional-ips/imports/import/import.controller.js

```diff
@@ -0,0 +1,47 @@
+const MESSAGES_CONTAINER_NAME = 'pci.projects.project.additional-ips';
+
+export default class AdditionalIpImportsImportController {
+  /* @ngInject */
+  constructor($state, $stateParams, $translate, CucCloudMessage, OvhApiIp) {
+    this.$state = $state;
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.OvhApiIp = OvhApiIp;
+
+    this.projectId = $stateParams.projectId;
+    this.serviceName = $stateParams.serviceName;
+
+    this.isLoading = false;
+  }
+
+  goBack() {
+    this.$state.go('^');
```

Comment:
better into routing.

Reply:
this is a legacy code moved to different location.